client/state/selectors/get-available-concierge-sessions.js

```diff
@@ -0,0 +1,7 @@
+import { get } from 'lodash';
+
+import 'calypso/state/concierge/init';
+
+export default ( state ) => {
+	return get( state, 'concierge.availableSessions', [] );
+};
```

Comment:
We're trying to remove `lodash` from the codebase, so we should use native Javascript for this. Also, we could use TypeScript for this new file.
```suggestion
import 'calypso/state/concierge/init';

export default ( state ) => {
	return state?.concierge?.availableSessions ?? [];
};
```